tests/Rector/MethodCall/WhereToWhereLikeRector/Fixture/Postgres/with_postgres_driver.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace RectorLaravel\Tests\Rector\MethodCall\WhereToWhereLikeRector\Fixture;
+
+use Illuminate\Database\Query\Builder;
+
+class Fixture
```

Comment:
It's best not to name classes in tests the same. I've found it causes weird issues. It's best to name the class WithPostgresDriver to match the file name.